docs/pywbemcli/cmdshelp.rst

```diff
@@ -190,14 +209,14 @@ Help text for ``pywbemcli class`` (see :ref:`class command group`):
       -h, --help  Show this message and exit.
 
     Commands:
-      associators   List the classes associated with a class.
-      get           Get a class.
       enumerate     List top classes or subclasses of a class in a namespace.
-      tree          Show the subclass or superclass hierarchy for a class.
-      references    List the classes referencing a class.
+      get           Get a class.
+      delete        Delete a class.
       invokemethod  Invoke a method on a class.
+      references    List the classes referencing a class.
+      associators   List the classes associated with a class.
       find          List the classes with matching class names on the server.
-      delete        Delete a class.
+      tree          Show the subclass or superclass hierarchy for a class.
```

Comment:
This reordering of commands should not be there. Please make sure you are using the latest master branch as a basis for this file.

Reply:
DONE

Reply:
The file is still showing the reordered commands. If you want to make sure it is re-created, delete it and then run make build or make builddoc.

Reply:
The order of commands depends on whether py2 or py3 is used to generate the file. Issue #510 has been opened for that. I suggest that for now, we use py2 to generate this file in any PRs.